packages/vanilla-components/package.json

```diff
@@ -36,6 +36,7 @@
     "intl-relativeformat": "^2.1.0",
     "marked": "^0.4.0",
     "path-to-regexp": "^2.2.1",
+    "progressbar.js": "^1.0.1",
```

Comment:
Looks like a good library -- but unfortunately it isn't actively developed.

@jorgecgll I want to double check that are you confident this is the best choice?